modules/end-user-guide/pages/microsoft-visual-studio-code-open-source-ide.adoc

```diff
@@ -0,0 +1,31 @@
+:_content-type: CONCEPT
+:description: Microsoft Visual Studio Code - Open Source IDE
+:keywords: user-guide, selecting, IDE, select, selecting, editor, ide, vs-code, visual-studio, visual-studio-code, jetbrains, intellij, pycharm, theia, che-theia
+:navtitle: Microsoft Visual Studio Code - Open Source
+:page-aliases:
+
+[id="microsoft-visual-studio-code-open-source-ide"]
+= Microsoft Visual Studio Code - Open Source IDE
+
+The {prod-short} build of link:https://github.com/microsoft/vscode[Microsoft Visual Studio Code - Open Source] is the default IDE of a new workspace.
+
+You can have extensions automatically installed in this IDE for workspaces, as explained in the next section, from the xref:administration-guide:extensions-for-microsoft-visual-studio-code-open-source.adoc[Open VSX registry].
```

Comment:
I'd get rid of the self-referential bit: "as explained in the next section"

Reply:
I actually don't think that this is a self-referential. It's referring to another section. Ideally, I would have named and linked the other section (as we cross-reference other sections elsewhere in the docs), but I don't think it's possible here. Moreover, it's just the next section, so this also makes it clear to the reader to just keep reading.